NeuroID/NeuroIDTracker.swift

```diff
@@ -24,7 +25,15 @@ public class NeuroIDTracker: NSObject {
         nidClassName = controller?.nidClassName
     }
     
+    
     public func captureEvent(event: NIDEvent) {
+        if (NeuroID.lowMemory){
+            if !NeuroID.isStopped() {
+                DataStore.insertCleanedEvent(event: NIDEvent(type: NIDEventName.lowMemory), storeType: "event")
+                // Try to send the events, send() will then stop the SDK to prevent further memory allocations
+                NeuroID.send()
+            }
```

Comment:
We capture a low memory event when the system alerts us, do we need to capture a second one? I think we should just try to send instead because the LM event will already exist